ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -60,15 +65,38 @@ data = yaml.load(open(path));
 data["compute"] = [ { "name": "worker", "replicas": 0 } ];
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 
+### Update base domain setting
+if [[ -v IS_XPN ]] && [[ -f "${CLUSTER_PROFILE_DIR}/ci-xpn.json" ]]; then
+  echo "Updating base domain (cluster profile 'gcp-qe')..."
+  python -c '
```

Comment:
personally, I prefer using `yq` instead of a piece of python code, less code and easy to read.

Reply:
Existing code in the file were using python, so I just followed up.